diff --git a/it87.c b/it87.c
index b8cbb47..218c106 100644
--- a/it87.c
+++ b/it87.c
@@ -659,7 +659,7 @@ static const struct it87_devices it87_devices[] = {
 		.features = FEAT_NEWER_AUTOPWM | FEAT_11MV_ADC | FEAT_16BIT_FANS
 		  | FEAT_TEMP_PECI | FEAT_FIVE_FANS
 		  | FEAT_FIVE_PWM | FEAT_IN7_INTERNAL | FEAT_PWM_FREQ2
-		  | FEAT_AVCC3 | FEAT_NEW_TEMPMAP,
+		  | FEAT_AVCC3 | FEAT_NEW_TEMPMAP | FEAT_BANK_SEL,
 		.num_temp_limit = 6,
 		.num_temp_offset = 6,
 		.num_temp_map = 6,