diff --git a/PKGBUILD b/PKGBUILD
index 6345de7..1eaf4b6 100644
--- a/PKGBUILD
+++ b/PKGBUILD
@@ -2,8 +2,8 @@

 _pkgbase=it87
 pkgname=it87-dkms-git
-pkgver=152.2b8b4fe
-pkgrel=3
+pkgver=287.4bff981
+pkgrel=1
 pkgdesc="Linux Driver for ITE LPC chips"
 arch=('x86_64' 'i686')
 url="https://github.com/frankcrawford/it87"
@@ -15,11 +15,13 @@ conflicts=('it87-frankcrawford-dkms-git')

 source=("$_pkgbase::git+https://github.com/frankcrawford/it87.git"
         "dkms.conf"
-        "it87.conf")
+        "it87.conf"
+        "odroid.patch")

 sha256sums=('SKIP'
             'f325b751c8a81416a75c2c1e7a7bc9ca46ae0fa3b44d4ccc09593274be1b2dc7'
-            'acdc488d1505e891ed6259b29428d4b27d26d18e3ea170f017b930390d6420e7')
+            'acdc488d1505e891ed6259b29428d4b27d26d18e3ea170f017b930390d6420e7'
+            'f21db635cfaf57e58a0b8595225673f5f8b66da02956bcb00c33806c8198d136')

 pkgver() {
   cd "$srcdir/$_pkgbase"
@@ -28,6 +30,7 @@ pkgver() {

 prepare() {
   cd "$srcdir/$_pkgbase"
+  patch -p1 -i "${srcdir}/odroid.patch"
 }

 package() {